web/templates/invite/facade.tmpl

```diff
@@ -6,13 +6,12 @@
 
   <a
     id="join-room-uri"
-    href="#"
-    data-href="{{.JoinRoomURI}}"
+    href="{{.JoinRoomURI}}"
```

Comment:
Using `href` instead of `data-href` allows users to right-click and "Copy link", and it seems that the JS behavior remains the same after this change (tested on desktop and mobile browsers).